### PR TITLE
es2016+

### PR DESCRIPTION
**Pull request by green1052**

### CODE REVIEW

**Review by piquark6046 — CHANGES_REQUESTED** [review]
First of all, it works on my side. [^1]

But, there are issues:
1. Please don't leave a commit message like https://github.com/List-KR/NamuLink/pull/10/commits/9fd474b2f3bcc91fc42077f98574f6f9e6316a35.
Also, reading [this article](https://djkeh.github.io/articles/How-to-write-a-git-commit-message-kor/) is a good idea.

2. I want to adjust and remove many tabs and whitespaces.
Do you agree with me?


By the way, do you have any Apple device? [^2]

[^1]: Mozilla Firefox and Google Chrome with Violentmonkey.
[^2]: It is not required. it is just a question for debugging.

**Comment by piquark6046** [comment]
Of course, you can do my changes request later.

**Comment by green1052** [comment]
1. i got it
2. i dont understand
u said 4 tab -> 2 tab?
i dont have apple deivce

File: NamuLink.user.js [hunk 1]
```diff
@@ -8,187 +8,165 @@
...
-// @version      1.6.3
+// @version      2.0.0
 // @author       PiQuark6046 and contributors
 //
 // @match        https://namu.wiki/*
 //
 // @description        NamuLink blocks the license-abused PowerLink advertisement on NamuWiki.
 // @description:ko     NamuLink는 나무위키에 있는 라이선스를 위반한 파워링크 광고를 차단합니다.
 //
-// @grant        unsafeWindow
```
[the remark this answers is not in the document]
Comment (green1052) [reply NamuLink.user.js:19]: no, adguard working without unsafeWindow perm

**Comment by green1052** [comment]
i prefer 4 tabs

**Comment by piquark6046** [comment]
OK. For now, I will use a native feature of Visual Studio Code.

**Review by piquark6046 — CHANGES_REQUESTED** [review]
@seia-soto Can you test your Safari environment?

**Comment by piquark6046** [comment]
https://github.com/List-KR/NamuLink/pull/10#issuecomment-1349130730
To read the result of calling `TypedArray.prototype.slice`.

**Comment by piquark6046** [comment]
https://github.com/List-KR/NamuLink/pull/10#issuecomment-1349147651
@green1052 

I don't recommend that.
Please read [this comment](https://github.com/List-KR/NamuLink/issues/7#issuecomment-1324150043).

**Comment by green1052** [comment]
> 

1. well i think it is bad for performance

3. https://nomo.asia/429

4. namu.wiki use only Uint8Array
if u want revert i revert that

**Comment by green1052** [comment]
so how to do?

**Comment by piquark6046** [comment]
First, please revert 3.

**Comment by green1052** [comment]
and than?

**Comment by piquark6046** [comment]
Finally, please revert no 1 unless. :D

**Comment by green1052** [comment]
i think better leave it

**Comment by piquark6046** [comment]
Then, do you have any experimental data?

**Comment by green1052** [comment]
i tested my code

**Comment by piquark6046** [comment]
What did you test? debugging test? Comparing the lead time of each code?

**Comment by green1052** [comment]
debugging

**Comment by piquark6046** [comment]
I had performed the debugging before i asked.
I want an experimental data.

**Comment by piquark6046** [comment]
https://github.com/List-KR/NamuLink/pull/10/commits/a00b20b1caf791310cd0d575adc5dd0191cbd268 ???

**Comment by Githubmaniak** [comment]
@piquark6046 잘 작동합니다

**Comment by green1052** [comment]
need more test?

**Review by piquark6046 — APPROVED** [review]

**Pull request merged**